demo/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.application.name=springboot-demo
+spring.datasource.url=jdbc:mysql://localhost:3306/webshop
+spring.datasource.username=root
+spring.datasource.password=dEb05121
```

Comment:
You have included application.properties with sensitive information - this is considered to be a very serious violation. Please remove this file

Reply:
application.properties is still there. please remove it